code/style.css

```diff
@@ -145,6 +160,31 @@ button {
   color: var(--primary);
 }
 
+/* START WINDOW */
+.start-wrapper {
```

Comment:
Very nice touch with a start window!
In desktop view, I can see two rows of dinos and white background when scrolling down. I suppose it doesn’t matter because you’re not ment to be scrolling, you are ment to click the start button. But if you want to improve this project then consider making the window go all the way down.

Reply:
Thank you for noticing that! Will take a look at it and fix it :)